diff --git a/CMakeLists.txt b/CMakeLists.txt
index d34b8f8..ea6d89f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -417,10 +417,8 @@ endif()
 set_target_properties(dark PROPERTIES OUTPUT_NAME "darknet")
 
 if(OpenCV_FOUND AND OpenCV_VERSION VERSION_GREATER "3.0" AND BUILD_USELIB_TRACK)
-  add_executable(uselib_track ${CMAKE_CURRENT_LIST_DIR}/src/yolo_console_dll.cpp)
 endif()
 
-add_executable(uselib ${CMAKE_CURRENT_LIST_DIR}/src/yolo_console_dll.cpp)
 if(BUILD_AS_CPP)
   set_target_properties(uselib PROPERTIES LINKER_LANGUAGE CXX)
 endif()
